ios/Classes/NativeImagesPlugin.m

```diff
@@ -1,37 +0,0 @@
-#import "NativeImagesPlugin.h"
-
-#if !__has_feature(objc_arc)
-#error ARC must be enabled!
-#endif
-
-@interface NativeImagesPlugin ()
-@end
-
-@implementation NativeImagesPlugin
-
-+ (void)registerWithRegistrar:(NSObject<FlutterPluginRegistrar> *)registrar {
-  FlutterMethodChannel *channel = [FlutterMethodChannel
-      methodChannelWithName:@"plugins.flutter.io/native_images"
-            binaryMessenger:[registrar messenger]];
-
-  [channel
-      setMethodCallHandler:^(FlutterMethodCall *call, FlutterResult result) {
-        if ([@"loadImage" isEqualToString:call.method]) {
-          NSString* name = call.arguments;
-          UIImage* image = [UIImage imageNamed:name];
```

Comment:
Add more validation steps so it's more debuggable for the user (i.e. is the whole thing null because I didn't pass in an image name vs UIImage can't find that image vs the format is unsupported etc)